src/core/components/rerender/Rerender.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState, type ReactNode } from 'react';
+
+interface RerenderProps {
```

Comment:
Just to follow current naming conventions,
```suggestion
export interface IRerenderProps {
```

Reply:
✅